webui/react/src/utils/error.ts

```diff
@@ -27,6 +27,7 @@ const openNotification = (e: DetError) => {
 
   notification?.({
     description: e.publicMessage || '',
+    key: e.name,
```

Comment:
i used `key` to identify the notification. 

https://ant.design/components/notification#components-notification-demo-update

Reply:
Cool, good to know